projects/plugins/social/src/js/components/admin-page/test/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import { useSelect, createReduxStore, register } from '@wordpress/data';
+import React from 'react';
+import { STORE_ID, storeConfig } from '../../../store';
+import Admin from '../index';
+
+const store = createReduxStore( STORE_ID, storeConfig );
+register( store );
+
+describe( 'load the app', () => {
+	test( 'container renders', () => {
+		const version = '99.9';
+		let storeSelect;
+		renderHook( () => useSelect( select => ( storeSelect = select( STORE_ID ) ) ) );
+		jest.spyOn( storeSelect, 'showPricingPage' ).mockReset().mockReturnValue( true );
+		jest.spyOn( storeSelect, 'hasPaidPlan' ).mockReset().mockReturnValue( true );
+		jest.spyOn( storeSelect, 'isShareLimitEnabled' ).mockReset().mockReturnValue( true );
```

Comment:
These calls are not particularly necessary.

Reply:
Do you mean that they're just examples?

Reply:
Yeah, kinda. I added it there while I was trying to figure out why the store was empty. Turns out it wasn't registered automatically, as in other plugins. Also, the above store keys are populated with default values, so there is no need to explicitly set them via `spyOn` calls in this "default" rendering case.